package.json

```diff
@@ -26,7 +26,7 @@
     "clean": "make clean",
     "help": "make help",
     "refresh": "make refresh",
-    "test": "make test"
+    "test": "mocha -R list"
```

Comment:
It was actually suggested to me by @rmg before we should actually not use make for npm test and use mocha directly for CI reasons. @rmg Can you comment here?